src/platform/plugins/shared/dashboard/server/content_management/v3/transforms/in/sections_in_transforms.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { v4 as uuidv4 } from 'uuid';
+
+import { DashboardSavedObjectAttributes } from '../../../../dashboard_saved_object';
+import { DashboardAttributes } from '../../types';
+
+export function transformSectionsIn(
+  sections: DashboardAttributes['sections']
+): DashboardSavedObjectAttributes['sections'] {
+  const updatedSections = sections?.map(({ id, order, ...rest }, index) => {
+    return {
+      ...rest,
+      order: order ?? index,
```

Comment:
_I swear I left this comment already, but it disappeared. Sorry if duplicate._

Doesn't the first item in the array have an `index = 0`? Does that break the constraint on the saved object schema?

Reply:
🤦 You're right - good catch. Fixed in 01e2c7712f49cf7dd998fa5a91a6ba852d53b7a2